cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.common.provider import cleanup_vm
+from cfme.infrastructure.provider import InfraProvider
+from cfme.services.catalogs.service_catalogs import ServiceCatalogs
+from cfme.services.requests import Request
+from cfme.services.myservice import MyService
+from cfme import test_requirements
+from utils.log import logger
+from utils import testgen
+from utils.version import current_version
+from utils.appliance import get_or_create_current_appliance, ViaSSUI
+
+
+pytestmark = [
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.usefixtures('vm_name', 'catalog_item', 'uses_infra_providers'),
+    test_requirements.ssui,
+    pytest.mark.long_running
+]
+
+
+pytest_generate_tests = testgen.generate([InfraProvider], required_fields=[
+    ['provisioning', 'template'],
+    ['provisioning', 'host'],
+    ['provisioning', 'datastore']
+], scope="module")
+
+
+@pytest.yield_fixture(scope="function")
+def order_catalog_item_in_ops_ui(provider, setup_provider, catalog_item, request):
+    """Tests order catalog item
+    Metadata:
+        test_flag: provision
+    """
+    vm_name = catalog_item.provisioning_data["vm_name"]
+    request.addfinalizer(lambda: cleanup_vm(vm_name + "_0001", provider))
+    catalog_item.create()
+    service_catalogs = ServiceCatalogs(catalog_item.catalog, catalog_item.name)
+    service_catalogs.order()
+    logger.info("Waiting for cfme provision request for service {}".format(catalog_item.name))
+    request_description = catalog_item.name
+    provision_request = Request(request_description, partial_check=True)
+    provision_request.wait_for_request()
+    assert provision_request.is_finished()
+    yield catalog_item.name
+
+
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+@pytest.mark.parametrize('context', [ViaSSUI])
+def test_myservice_crud(context, order_catalog_item_in_ops_ui):
+    appliance = get_or_create_current_appliance()
+    service_name = order_catalog_item_in_ops_ui
+    with appliance.context.use(context):
+        appliance.server.login()
+        myservice = MyService(appliance, service_name)
+        myservice.update({'name': service_name + '_edited'})
```

Comment:
Prefer ``format`` for string concatenation.

Reply:
Done